corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -715,23 +716,28 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
           </div>
         </div>
       </div>
-      <div class="modal-footer">
-        {% if can_edit_roles %}
-          <button type="button"
-                  class="btn btn-default"
-                  data-bind="click: $root.unsetRoleBeingEdited">
-            {% trans "Cancel" %}
-          </button>
-          <div data-bind="saveButton2: $root.modalSaveButton.state,
-                                    saveOptions: $root.modalSaveButton.saveOptions"></div>
-        {% else %}
-          <button type="button"
-                  class="btn btn-default"
-                  data-bind="click: $root.unsetRoleBeingEdited">
-            {% trans "Close" %}
-          </button>
-        {% endif %}
-      </div>
+      <form novalidate data-bind="submit: $root.submitNewRole">
```

Comment:
what does the `novalidate` do here?

Reply:
It was used here: https://github.com/dimagi/commcare-hq/blob/591ede5b97ec68c6dca2c6db6706beda926c1039/corehq/apps/users/templates/users/mobile_workers.html#L88

So I just copied that. It seems to prevent native validation of form elements.